OSDOCS-2031: Adding known issue for customizing console routes (#33783)
openshift/openshift-docs: release_notes/ocp-4-8-release-notes.adoc

* The installation of {op-system} on a {op-system-base} KVM installation on IBM Z fails when using an ECKD type DASD as a VirtIO block device. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1960485[*BZ#1960485*])

* An Open Virtual Network (OVN) bug causes persistent connectivity issues with Octavia load balancers. When Octavia load balancers are created, OVN might not plug them into some Neutron subnets. These load balancers might be unreachable for some of the Neutron subnets. This problem affects Neutron subnets, which are created for each OpenShift namespace, at random when Kuryr is configured. As a result, when this problem occurs the load balancer that implements OpenShift `Service` objects will be unreachable from OpenShift namespaces affected by the issue. Because of this bug, {product-title} 4.8 deployments that use Kuryr SDN are not recommended on {rh-openstack-first} 16.1 with OVN and OVN Octavia configured until the bug is fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937392[*BZ#1937392*])

> * When either of the console's routes (`console` or `download`) are customized, the Console Operator does not properly update the `Ingress` resource with the `componentRoutes` conditions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954148[*BZ#1954148*])

Reviewer: the Console Operator does not properly update the `Ingress` resource with the `componentRoutes` conditions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954148[*BZ#1954148*])
```suggestion
* The Console Operator does not properly update the `Ingress` resource with the `componentRoutes` conditions for  either of the console's routes (`console` or `download`). (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954148[*BZ#1954148*])
```

Reviewer: Also - [it looks like](https://deploy-preview-33783--osdocs.netlify.app/openshift-enterprise/latest/web_console/customizing-the-web-console.html#customizing-the-download-route_customizing-web-console) the "download route" is called the "downloads".
